File: src/main/java/com/finki/timesheets-client/src/app/model/ProjectPositionsDto.ts
import {Project} from "./Project";
import {WorkPackage} from "./WorkPackage";

export class ProjectPositionsDto {
  id: number;
  project: Project;
  positions: any;

  constructor(project: Project, positions: any, workPackage: WorkPackage) {
    this.project = project;
    this.project.workPackage = workPackage;
    this.positions = positions;
  }
}

File: src/main/java/com/finki/timesheets/model/dto/ProjectPositionsDto.java
package com.finki.timesheets.model.dto;

import com.finki.timesheets.model.Project;

import java.util.List;

;


public class ProjectPositionsDto {
    private Project project;
    private List<PositionSalaryDto> positions;

    public ProjectPositionsDto() {
    }



    public Project getProject() {
        return project;
    }

    public void setProject(Project project) {
        this.project = project;
    }


    public List<PositionSalaryDto> getPositions() {
        return positions;
    }

    public void setPositions(List<PositionSalaryDto> positions) {
        this.positions = positions;
    }
}
